### ISSUE TITLE
Does it support Outlook?

### ISSUE THREAD

**Issue by hanifmhd** [vonovak/react-native-add-calendar-event#121]
Hi guys,

I wanna ask, does it support Outlook calendar? I don't see the example of outlook in readme file.

Thank you.

**Comment by Kalpesh1Ghori** [comment]
Yes..Please install OutlookCalendar from PlayStore or AppStore so, After that you can see and select calendar from bottom presentEventCreatingDialog from device.
